packages/block-library/src/social-links/edit.js

```diff
@@ -45,6 +45,7 @@ const getDefaultBlockLayout = ( blockTypeOrName ) => {
 };
 
 export function SocialLinksEdit( props ) {
+	const [ backgroundBackup, setbackgroundBackup ] = useState( {} );
```

Comment:
```suggestion
	const backgroundBackup = useRef( {} );
```
It's not crucial but I've realized `useRef` may be better here since another render is not needed when the value changes. Besides that, I'd put this on a line that's closer to the hook it's used in as that seems more consistent with the style of the rest of the codebase.